lib/rspec/support/ruby_features.rb

```diff
@@ -45,7 +45,7 @@ def mri?
     # Provides query methods for ruby features that differ among
     # implementations.
     module RubyFeatures
-      module_function
+    module_function
```

Comment:
I personally prefer this indented, can we modify the rule instead? /cc @rspec/owners